noir-projects/noir-protocol-circuits/crates/types/src/meta/mod.nr

```diff
@@ -72,153 +66,138 @@ pub comptime fn generate_deserialize_from_fields<N>(
     typ: Type,
     field_array_name: Quoted,
     num_already_consumed: u32,
-    to_replace: Quoted,
-    ro_replace_with: Quoted,
     should_unpack: bool,
 ) -> (Quoted, u32) {
     let mut result = quote {};
     // Counter for the number of fields consumed
     let mut consumed_counter: u32 = 0;
 
-    // Check if there is a replacement for the currently processed field
-    if name == to_replace {
```

Comment:
Removing the `if {} else {}` here resulted in the large diff as after formatting all the lines were moved left by 1 tab. There are no other changes here other then nuking the `to_replace` and `to_replace_with` args.

Reply:
Thanks for pointing this out, the diff is indeed saner when hiding whitespace differences.